### PR TITLE
Sprint 11 translations

### PR DESCRIPTION
**Pull request by sergei-maertens**
- [x] Javascript
- [x] Django

### CODE REVIEW

**Review by joeribekker — APPROVED** [review]

**Comment by joeribekker** [comment]
@sergei-maertens might want to double check, I cannot assign you as reviewer.

File: src/openforms/conf/locale/nl/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -77,6 +77,24 @@ msgstr "gebruiker"
...
+
+#: openforms/api/drf_spectacular/hooks.py:18
+msgid ""
+"Amount of time in seconds after which the session expires. After this time "
+"has passed, the session is expired and the user is 'logged out'. Note that "
+"every subsequent API call resets the expiry."
+msgstr ""
+"Aantal seconden waarna een sessie verloopt. Na het verlopen van de sessie is "
+"de gebruiker uitgelogd en kan zijn huidige inzending niet meer afmaken. "
+"Opmerking: Bij elke interactie met de API wordt de sessie verlengt."
```
Comment (sergei-maertens) [inline src/openforms/conf/locale/nl/LC_MESSAGES/django.po]:
```suggestion
"Opmerking: Bij elke interactie met de API wordt de sessie verlengd."
```

File: src/openforms/conf/locale/nl/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -554,29 +571,25 @@ msgid "DigiD Mock"
 msgstr "DigiD simulatie"
 
 #: openforms/authentication/contrib/eherkenning/apps.py:8
-msgid "eHerkenning authentication plugin"
-msgstr "eHerkenning authenticatieplugin"
+msgid "eHerkenning/eIDAS authentication plugin"
+msgstr "eHerkenning/eIDAS authenticatie plugin"
```
Comment (sergei-maertens) [inline src/openforms/conf/locale/nl/LC_MESSAGES/django.po]:
```suggestion
msgstr "eHerkenning/eIDAS authenticatieplugin"
```

File: src/openforms/conf/locale/nl/LC_MESSAGES/django.po [hunk 3]
```diff
@@ -778,7 +799,11 @@ msgstr "Sessies"
 msgid "Data removal"
 msgstr "Gegevens schonen"
 
-#: openforms/config/admin.py:118
+#: openforms/config/admin.py:117
+msgid "Plugin configuration"
+msgstr "Plugin configuratie"
```
Comment (sergei-maertens) [inline src/openforms/conf/locale/nl/LC_MESSAGES/django.po]:
```suggestion
msgstr "Pluginconfiguratie"
```

File: src/openforms/conf/locale/nl/LC_MESSAGES/django.po [hunk 4]
```diff
@@ -1038,230 +1068,253 @@ msgstr ""
...
+#: openforms/config/models.py:436
 msgid "Amount of days when all submissions will be permanently deleted"
 msgstr ""
 "Aantal dagen dat een inzending bewaard blijft voordat deze definitief "
 "verwijderd wordt."
 
-#: openforms/config/models.py:413
+#: openforms/config/models.py:440
+msgid "plugin configuration"
+msgstr "plugin configuratie"
```
Comment (sergei-maertens) [inline src/openforms/conf/locale/nl/LC_MESSAGES/django.po]:
```suggestion
msgstr "pluginconfiguratie"
```

File: src/openforms/conf/locale/nl/LC_MESSAGES/django.po [hunk 5]
```diff
@@ -3696,14 +3879,228 @@ msgstr ""
...
+
+#: openforms/registrations/contrib/camunda/serializers.py:20
+#: openforms/registrations/contrib/camunda/serializers.py:159
+msgid "Only enabled variables are passed into the process"
+msgstr "Alleen ingeschakelde variabelen worden doorgegeven aan het proces"
+
+#: openforms/registrations/contrib/camunda/serializers.py:25
+#: openforms/registrations/contrib/camunda/serializers.py:96
+msgid "Component key"
+msgstr "Component sleutel"
```
Comment (sergei-maertens) [inline src/openforms/conf/locale/nl/LC_MESSAGES/django.po]:
```suggestion
msgstr "Componentsleutel"
```

File: src/openforms/conf/locale/nl/LC_MESSAGES/django.po [hunk 6]
```diff
@@ -3696,14 +3879,228 @@ msgstr ""
...
+#: openforms/registrations/contrib/camunda/serializers.py:143
+msgid "The list with nested variable definitions"
+msgstr "De lijst met geneste variabeledefinities"
+
+#: openforms/registrations/contrib/camunda/serializers.py:166
+msgid ""
+"Name of the variable in the Camunda process instance. For complex variables, "
+"the name must be supplied."
+msgstr ""
+"Naam van de variabele in het Camunda proces. For complexe variabelen moet "
```
Comment (sergei-maertens) [inline src/openforms/conf/locale/nl/LC_MESSAGES/django.po]:
```suggestion
"Naam van de variabele in het Camunda proces. Voor complexe variabelen moet "
```

File: src/openforms/js/lang/formio/nl.json [hunk 7]
```diff
@@ -165,7 +165,7 @@
   "BSN Field": "BSN",
   "Postcode Field": "Postcode",
   "File Upload": "Bestandsupload",
-  "Text Area": "Textvlak",
+  "Text Area": "Tektvlak",
```
Comment (sergei-maertens) [inline src/openforms/js/lang/formio/nl.json:168]:
```suggestion
  "Text Area": "Tekstvlak",
```

File: src/openforms/js/lang/formio/nl.json [hunk 8]
```diff
@@ -236,9 +236,43 @@
...
+  "This component should Display:": "Dit component moet worden weergegeven",
+  "When the form component:": "Als het formulier component",
+  "Has the value:": "De waarde heeft:",
+  "When this is checked, the image will be resized.": "Wanneer dit is ingeschakeld wordt het afbeeldingsformaat gewijzigd.",
+  "Simple": "Eenvoudig",
+  "Paragraph": "Paragraaf",
+  "Refresh On Change": "Vernieuwen bij wijzigingen",
+  "File Types": "Bestandstypen",
+  "The maximum number of decimal places.": "Het maximaal aantal decimalen.",
+  "Column Properties": "Kolom eigenschappen",
```
Comment (sergei-maertens) [inline src/openforms/js/lang/formio/nl.json:273]:
```suggestion
  "Column Properties": "Kolomeigenschappen",
```

**Pull request merged**